### ISSUE TITLE
ETQ admin, je souhaite que mon DPO soit informé de la création d’une démarche en ligne

### ISSUE THREAD

**Issue by emsnytech** [betagouv/tps#3485]
Bonne remarque d’un utilisateur, qui rejoint l’une des préoccupations qu’avait soulevé le DPO du SPM.
A savoir que le DPO de l’organisme soit informé de la mise en production d’une démarche.
https://secure.helpscout.net/conversation/783582978/12674?folderId=1653799

On peut aujourd’hui répondre que l’agent doit faire cette démarche.
On peut rajouter dans l’interface de création d’un service une phrase pour inciter l’administrateur à informer le DPO de la mise en place de la démarche.
Plus tard, on pourrait rajouter un champs en effet lors de la publication d’une démarche, par exemple avec «  adresse mail du DPO » et faire un bouton avec un mail préformaté envoyé au DPO avec des éléments de la description de la procédure.

**Comment by kemenaran** [comment]
Effectivement.

Cela dit je serais bien pour en faire quelque chose d'incitatif et facultatif au moment de la création de la démarche, pour ne pas mettre des bâtons dans les roues de gens qui veulent juste tester à quoi DS ressemble.

Et effectivement, au moment de la publication, on peut insister un peu plus 👍

**Comment by maatinito** [comment]
Est-ce que ce champ ne devrait pas être dans la description du service.

Etant donné qu'il est maintenant possible de tester sans problème une 
démarche, le fait de spécifier l'adresse de mail du DPO dans le service 
déclencherait automatiquement un envoi de mail lors d'une publication.

Et si rien n'a été mis, aucun mail ne serait envoyé et cela permettrait 
de tester ses démarches.

Le 20/02/2019 à 07:55, emsnytech a écrit :

**Comment by kemenaran** [comment]
@maatinito oui, complètement. On avait deux issues dupliquées, je viens de les fusionner.

Le plan pour une version 0 serait :

* On ajoute un champ **optionnel** "Adresse mail du DPO" à la page Service, avec une explication claire sur le rôle du DPO. _Comme ça on sensibilise les admins au rôle du DPO, sans ralentir la création de démarche._
* (Au passage on supprime le champ SIRET de la page service)

Et plus tard pour une v1 :

* On envoie un mail automatiquement au DPO avant la publication, ou à la publication.
